.changeset/curvy-bears-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/theme': patch
+---
+
+Added the `no-top-level-material-ui-4-imports` ESLint rule to aid with the migration to Material UI v5
```

Comment:
```suggestion
Internal refactor to avoid top-level imports from MUI.
```

I think the current message might be a bit confusing to consumers of this package, since the changes are internal to the package.

Reply:
sure, I think whatever the description will be for that, someone will confuse it anyway ;)